### ISSUE TITLE
Issues with Space Boundaries on Quest Pro When Running Unity 2022.3.1 Build

### ISSUE THREAD

**Issue by sakusan393** [oculus-samples/Unity-Discover#6]
### Environment
- Unity Version: 2022.3.1 or 2022.3.4
- Quest Pro (Runtime Version:57.0.0.261.669.517337457)

### Description
I'm experiencing an issue related to space boundaries on Oculus Quest Pro. Despite having completed the Physical Space setup in the Quest Pro settings, the space boundaries disappear when the content starts, making it impossible to place objects within the space.

### Steps to Reproduce
1. Clone the project from GitHub.
2. Change the platform to Android.
3. Select the Discover scene.
4. Enter the Photon Fusion ID and Oculus/Platform App ID.
5. Build the APK and run it on Quest Pro.

### Expected Behavior
The space boundaries should be visible, and objects should be able to be placed within the content.

### Observed Behavior
The space boundaries disappear when the APK is run on the Quest Pro.

### Additional Information
- When connecting the Quest Pro to Unity Editor using Quest Link and running the scene, the space boundaries are correctly displayed.
- The Discover app installed from App Lab worked correctly. The issue occurs after uninstalling the App Lab version and running the APK built from the GitHub source.
- Here's a YouTube video for further clarification:
[![YouTube Video](http://img.youtube.com/vi/uUf5u0te2cY/0.jpg)](https://www.youtube.com/watch?v=uUf5u0te2cY)

**Comment by sakusan393** [comment]
### Issue Resolved

I've successfully resolved the issue. Following the guidance from [this forum post](https://forum.unity.com/threads/introducing-meta-quest-support-in-ar-foundation.1450966/page-2#post-9325793), I added `<uses-permission android:name="com.oculus.permission.USE_SCENE"/>` to the AndroidManifest.xml. This triggered a confirmation dialog, and as a result, the Space boundaries became active.

#### Steps to Resolve
1. Open AndroidManifest.xml in your project.
2. Add the line `<uses-permission android:name="com.oculus.permission.USE_SCENE"/>`.
3. Rebuild the APK and run it on Quest Pro.

This should prompt a dialog for permission, and once confirmed, the space boundaries should function correctly.

**Comment by kingdomax** [comment]
@sakusan393, I faced the same issue as you did, where my built APK app couldn't detect any scene models (walls, floors, tables, etc.). However, when I run the app directly via Unity's play mode (i.e., Quest Link), it works fine. Thanks to your suggestion, the problem is now resolved.

@AlexThiv, I think this issue should be addressed in the document, as people might build on top of this project, and I have seen this problem occur several times (e.g. #8 #9).
